BothamUI/BothamPullToRefreshPresenter.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  BothamPullToRefreshPresenter.swift
+//  BothamUI
+//
+//  Created by Davide Mendolia on 26/11/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+
+public protocol BothamPullToRefreshPresenter: BothamPresenter {
+    func refresh(didRefresh: () -> ())
```

Comment:
Shouldn't we use something more descriptive? Like ``onPullToRefresh`` or something similar?